src/container/getRelayQueries.js

```diff
@@ -54,7 +54,8 @@ function getRelayQueries(
       var concreteQuery = buildRQL.Query(
         queryBuilder,
         Component,
-        Object.keys(route.params)
+        Object.keys(route.params),
+        Component.getFragment(fragmentName)
```

Comment:
It's important to pass the route params by default to the fragment, as the user can't manually do so otherwise.

```
Component.getFragment(fragmentName, route.params)
```